FinSearchUnified/Bundle/SearchBundleFindologic/ResponseParser/ResponseParser.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace FinSearchUnified\Bundle\SearchBundleFindologic\ResponseParser;
+
+use FINDOLOGIC\Api\Responses\Response;
+use FINDOLOGIC\Api\Responses\Xml21\Xml21Response;
+use FinSearchUnified\Bundle\SearchBundleFindologic\ResponseParser\QueryInfoMessage\QueryInfoMessage;
+use InvalidArgumentException;
+
+abstract class ResponseParser
+{
+    /**
+     * @var Xml21Response
```

Comment:
I have overseen this last review. This obviously should also be just a `Response`.